python/Produce_Config_File.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+import xlrd
+from array import array
+import numpy as np
+from ROOT import TGraph
+from ROOT import TMultiGraph
+import ROOT as r
+import argparse
+from optparse import OptionParser
+from options import parser
+import sys
+
+#parser = argparse.ArgumentParser(add_help=True)
+#parser.add_argument('file', nargs='+')
+#parser.add_argument('--test',action="store",dest="test",help="Select the type of test: QC3, QC4_HV QC4_SS")
+if len(sys.argv)==1:
+    parser.print_help()
+    sys.exit(1)
+
+
+
+
+def Color(j):
+	if j % 7 == 0:
+		ret_int='kRed'
+	elif j % 7 == 1:
+		ret_int='kRed+2'
+	elif j % 7 == 2:
+		ret_int='kRed+3'
+	elif j % 7 == 3:
+		ret_int='kBlue'
+	elif j % 7 == 4:
+		ret_int='kBlue+2'
+	elif j % 7 == 5:
+		ret_int='kGreen'
+	elif j % 7 == 6:
+		ret_int='kGreen+2'
+	
+
+	return ret_int
+
+
+				
+
+args = parser.parse_args()
+
+
+
+
+
+text_file = open(str(args.OutputName)+".cfg", "w")
+j=0
+
+text_file.write("[BEGIN_CANVAS]\n")
+text_file.write("    Canv_Axis_NDiv = '"+str(args.AxisNDiv)+"'; #X,Y\n")
```

Comment:
Lines 55 - 79, use the `\t` character to insert a tab, this will improve readability rather than manually inserting the spaces shown.